dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -173,47 +174,33 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
                                 S_OK,
                                 PushNotificationChannelStatus::CompletedSuccess);
                         }
-                        else if (hr == E_NOTIMPL)
-                        {
-                            usingLegacyImplementation = true;
-
-                            PushNotificationChannelManager channelManager{};
-                            winrt::PushNotificationChannel pushChannelReceived{ nullptr };
-
-                            pushChannelReceived = co_await channelManager.CreatePushNotificationChannelForApplicationAsync();
-
-                            PushNotificationTelemetry::ChannelRequestedByApi(S_OK, remoteId, usingLegacyImplementation);
-
-                            co_return winrt::make<PushNotificationCreateChannelResult>(
-                                winrt::make<PushNotificationChannel>(pushChannelReceived),
```

Comment:
We no longer hang of the wrapped PushNotificationChannel. All usages of it needs to be deleted.